app/controllers/application_controller.rb

```diff
@@ -1,7 +1,22 @@
+# class ApplicationController < ActionController::Base
+#   add_flash_types :danger, :info, :warning, :success, :messages, :notice, :alert
+# end
```

Comment:
- [ ] Committing old pieces of codes as inline comments makes your project looks messy. Please let's avoid doing this. If you need to review a previous version of your code, you can always use git history.